tests/test_gitbook.py

```diff
@@ -17,55 +18,56 @@ def _path(d):
 
 sys.path.append('.')
 
-def doctest_single(md_file):
-    orig_path = os.getcwd()
-    os.chdir(_path('.'))
-    try:
-        claripy.ast.base.var_counter = itertools.count()
-        lines = open(md_file, encoding='utf-8').read().split('\n')
-        test_enabled = False
-        multiline_enabled = False
-        multiline_stuff = ''
-        env = {}
+# pylint: disable=missing-class-docstring, consider-using-with, raise-missing-from, no-self-use
```

Comment:
Can you fix the issues so that `consider-using-with` and `raise-missing-from` can be re-enabled?